frontend/src/pages/devices/List.tsx

```diff
@@ -70,6 +70,7 @@ export const List = () => {
                     <tr>
                         <th scope="col">Name</th>
                         <th scope="col">Status</th>
+                        <th scope="col">Battery</th>
```

Comment:
These don't look like the right changes, you might need to merge `origin/main` into your branch.